resources/function_help/json/relate

```diff
@@ -32,7 +32,7 @@
     }],
     "examples": [{
       "expression": "relate( geom_from_wkt( 'LINESTRING(40 40,120 120)' ), geom_from_wkt( 'LINESTRING(40 40,60 120)' ), '**1F001**' )",
-      "returns": true
+      "returns": TRUE
```

Comment:
Question: this should probably be 
```suggestion
      "returns": "TRUE"
```
Or am I missing something?

Reply:
You are right, It could be `"returns": true` as before, or `"returns": "TRUE"`, while `"returns": TRUE` is invalid (literal values as `true` and `false` should be lowercase in JSON).
In fact the test are failing at such line: `json.decoder.JSONDecodeError: Expecting value: line 35 column 18 (char 1332)`

Reply:
You are right, It could be `"returns": true` as before, or `"returns": "TRUE"`, while `"returns": TRUE` is invalid (literal values as `true` and `false` should be lowercase in JSON).
In fact the test are failing at such line: `json.decoder.JSONDecodeError: Expecting value: line 35 column 18 (char 1332)`